build/deps/gen/dep_cxxbridge_cmd.bzl

```diff
@@ -2,11 +2,11 @@
 
 load("@//:build/http.bzl", "http_archive")
 
-URL = "https://crates.io/api/v1/crates/cxxbridge-cmd/1.0.138/download"
```

Comment:
We may have to freeze this for now – based on https://github.com/dtolnay/cxx/issues/1436, higher cxxbridge-cmd versions appear to require LLVM 17 (or maybe there's some compile flag issue). Updating the minimum LLVM version is something we can do, but it merits discussion/should be done separately.